pythia/common/defaults/configs/base.yml

```diff
@@ -1,5 +1,7 @@
 # Configuration for training
 training_parameters:
+    # Name of the trainer class used to define the training/evalution loop
+    trainer_name: 'base_trainer'
```

Comment:
Change this key to `trainer` instead of `trainer_name`? Rest looks good to me.

Reply:
Fixed in 294a0fa